requirements-docs.txt

```
@@ -20,7 +20,7 @@ charset-normalizer==3.3.2
 click==8.1.7
 colorama==0.4.6
 craft-application==4.2.7
-craft-archives==2.0.0
+craft-archives @ git+https://github.com/canonical/craft-archives@work/CRAFT-3538-noble-key-conflict
```

Just putting this here to make it easy if you take my suggestion about releasing today.

```suggestion
craft-archives==2.0.1
```